pkg/operator/starter.go

```diff
@@ -71,6 +74,30 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		configInformers,
 		ctx.EventRecorder,
 	)
+	resourceSyncController := resourcesynccontroller.NewResourceSyncController(
+		staticPodOperatorClient,
+		map[string]informers.SharedInformerFactory{
+			userSpecifiedGlobalConfigNamespace:    kubeInformersForUserSpecifiedGlobalConfigNamespace,
+			machineSpecifiedGlobalConfigNamespace: kubeInformersForMachineSpecifiedGlobalConfigNamespace,
+			operatorNamespace:                     kubeInformersForOpenshiftKubeAPIServerOperatorNamespace,
+			targetNamespaceName:                   kubeInformersForOpenshiftKubeAPIServerNamespace,
+			"kube-system":                         kubeInformersForKubeSystemNamespace,
+		},
+		kubeClient,
+		ctx.EventRecorder,
+	)
+	if err := resourceSyncController.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: etcdNamespaceName, Name: "etcd-serving-ca"},
```

Comment:
order is dest, src. We should standardize on one. Library-go's sync uses src, dest.

Reply:
> order is dest, src. We should standardize on one. Library-go's sync uses src, dest.

https://golang.org/pkg/builtin/#copy